getting-started/glossary.md

```diff
@@ -1,51 +1,93 @@
+Refers to the API credentials, required to generate an authorization token to make API calls.
+
+## CFT
+Cloud File Transfer. A centralized, fully managed file transfer service for the whole of government (WOG). CFT is a part of the SG Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).
 
-A public identifier for applications. Required to generate an authorization token to make API calls.
+## CDR
+Content Disarm and Reconstruct also known as Threat Extraction, proactively protects against known and unknown threats contained in documents by removing executable content.
 
-## Cloud File Transfer \(CFT\)
-A lightweight, fully managed, API-driven service that allows agencies or businesses to transfer files easily, securely and efficiently at scale.
+## GCC
+The Government Commercial Cloud Service brings the modern innovations and capabilities of commercial cloud computing platforms to less sensitive Government systems. These leading ICT capabilities are augmented by robust cybersecurity measures and systems to protect the data that resides on commercial cloud platforms.
 
-## Content Disarm and Reconstruct \(CDR\)
+
+## HTTPS
+Hypertext Transfer Protocol Secure is an extension of the Hypertext Transfer Protocol. It is used for secure communication over a computer network, and is widely used on the Internet.
+
+## IM8
+Instruction Manual 8. A management tool used in the Government to safeguard ICT & SS (Infocomm Technology and Smart Systems) assets and guides all Government agencies in adopting ICT & SS to enable their digital transformation.
 
 
 ## JWT
-JSON Web Token (JWT) is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
+JSON Web Token is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
+
+## KMS
+Key Management Service by AWS helps you create and manage cryptographic keys and control their use across a wide range of AWS services and in your applications.
+
+## Malware Scanning
+The process of detecting malware in a file.
 
 ## OpenAPI Specifications
 The OpenAPI Specification (OAS) defines a standard, language-agnostic interface to RESTful APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection.
 
 ## Payload
 The body of an API request or response message.
 
-## Project Id
+## PGP
+Pretty Good Privacy is an encryption program that provides cryptographic privacy and authentication for data communication. 
 
 ## REST API
 A REST API is an API that conforms to the design principles of the REST, or representational state transfer architectural style.
 
 ## Sandbox
 CFT sandbox environment is a secure environment to try out CFT APIs. The sandbox supports file transfers in the internet zone between systems.
```

Comment:
"The CFT sandbox is a secure test environment for you to try out our CFT APIs for file transfers in the internet zone between systems. CFT APIs for intranet zone transfers are not available for testing within the 
sandbox. "